format/postgres/pg_btree.md

```diff
@@ -0,0 +1,14 @@
+### Btree index meta page
+
+```sh
+$ fq -d pg_btree -o flavour=postgres14 ".[0] | d" 16404
+```
+
+### Btree index page
+
+```sh
+$ fq -d pg_btree -o flavour=postgres14 ".[1]" 16404
+```
+
+### References
+- https://www.postgresql.org/docs/current/storage-page-layout.html
```

Comment:
If you want you can add a authors section to all the *.md files, see bson.md etc (and do `make doc` afterwards)

Reply:
It's done.